ignite/handlers/checkpoint.py

```diff
@@ -259,10 +262,14 @@ def _check_lt_n_saved(self, or_equal=False):
 
     def __call__(self, engine: Engine) -> None:
 
-        suffix = ""
+        filename_pattern_dict = {
```

Comment:
Can we initialize it empty, for example: `filename_pattern_dict = {}` ?

Reply:
```suggestion
        filename_pattern_dict = {
            "filename_prefix": self.filename_prefix,
            "ext": self._ext,
            "global_step": "",
            "score_name": "",
            "score": "",
        }
```
@vfdev-5 is this ok or should I remove the blank ones. I kept the blank key-value pairs to avoid the key error.